xbmc/interfaces/legacy/Player.h

```diff
@@ -449,6 +449,21 @@ namespace XBMCAddon
       bool isPlayingRDS();
 #endif
 
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_Player
+      /// @brief \python_func{ isExternalPlayer() }
+      ///-----------------------------------------------------------------------
+      /// Check for external player.
+      ///
+      /// @return                    True if kodi is playing using an
+      ///                            external player.
+      ///
```

Comment:
could you please add a note like this one:
https://github.com/xbmc/xbmc/blob/e53988394230379d0e00ab1812fc3920fdaa6723/xbmc/interfaces/legacy/Control.h#L1106

so we can keep track on in which kodi version new functions were introduced.

Reply:
Sure thing!